js/okcoinusd.js

```diff
@@ -623,17 +623,29 @@ module.exports = class okcoinusd extends Exchange {
         return { 'url': url, 'method': method, 'body': body, 'headers': headers };
     }
 
-    async request (path, api = 'public', method = 'GET', params = {}, headers = undefined, body = undefined) {
-        let response = await this.fetch2 (path, api, method, params, headers, body);
+    handleErrors (code, reason, url, method, headers, body) {
+        if (this.verbose)
+            console.log (this.id, method, url, code, reason, body ? ("\nResponse:\n" + body) : '');
+        let response = JSON.parse (body);
+        if ('error_code' in response) {
+            if (this.errorCodes === undefined) {
+                this.errorCodes = {
+                    '1009': OrderNotFound,
+                    '1003': InvalidOrder,      // no order type (was left by previous author)
```

Comment:
Python PEP8 standard demands that here should be exactly 2 spaces before the in-line comment starts. In JS we only leave one space, and the second space is added by the transpiler to the Python-version of the source.
So, these lines should look like 
```
                         ↓
    '1003': InvalidOrder, // no order type (was left by previous author)
```

Transpiler is in fact very happy, it's the python code syntax check that fails.

Also, I think it may be a good idea to move these errorCodes to `describe()` as that method looks like appropriate place for initializing this property, what do you think?

Reply:
> Also, I think it may be a good idea to move these errorCodes to describe() as that method looks like appropriate place for initializing this property, what do you think?

Actually, I did it in the `describe()` from start, as it was *the right* thing to do. But then, from the practical perspective, I moved it closer to the usage site as it was to unhandy to jump between very end and the beginning of the file.

Reply:
We are free to move the method itself higher up in the file )